release_tools/github.py

```diff
@@ -88,6 +89,25 @@ def download_archive(self, branch, save_to_path, ball="zipball"):
             archive.extractall(save_to_path)
             print "Extracted"
 
+    def download_release_history(self, path):
+        url = "https://api.github.com/repos/{owner}/{repo}/releases{token}"\
+              .format(owner=self.owner, repo=self.repo, token=self.access_token_postfix())
+        response = requests.get(url)
+        if response.status_code == 200:
+            print "Writing to file..."
```

Comment:
It wouldn't hurt if this was changed to the print function (`from __future__ import print_function`) so it's ready for py3. Or change it to `logging.info(...)`?